### ISSUE TITLE
Section for page statistics

### ISSUE THREAD

**Issue by tobimori** [floriankarsten/kirby-plausible#9]
I'd be cool to have a section that can be added to a page blueprint to show the viewcount on the page directly. 
Specify the URL to look for with query language and having a sensible default using the previewUrl

**Comment by tobimori** [comment]
yeah exactly. quite a bit more work than what this addon does right now

**Comment by iskrisis** [comment]
It would make sense. I am just a bit hesitant because while we use plausible - they increased their prices a lot recently and i am not sure if its not in a bit of a downturn.

Imho when doing stuff like that it would make maybe more sense to do this as general analytics plugin with providers for api data for any kind of services. Even this plugin should have been general iframe handler. It adds only few aesthetic parameters specifically to Plausible.

**Comment by tobimori** [comment]
I thought about developing a generic plugin with a middleware-like infrastructure but the market is just too small in the Kirby universe to justify development of something like this tbh

I'm self hosting plausible and to me it's still the best choice in that regard, so if you end up leaving and need a maintainer I can take over.
